Sources/LogicKit/Term.swift

```diff
@@ -111,12 +180,12 @@ extension Term: CustomStringConvertible {
             return "\(value)"
         case let ._term(name, arguments):
             return arguments.isEmpty
-                ? name
-                : "\(name)(\(arguments.map({ $0.description }).joined(separator: ", ")))"
+                ? name.description
+                : "\(name.description)[\(arguments.map({ $0.description }).joined(separator: ", "))]"
```

Comment:
Swift's String interpolation calls for `.description` automatically, there's no need to add it explicitly.

Reply:
Fixed